packages/trigger/src/lib/excel-parser.ts

```
@@ -0,0 +1,192 @@
+	console.log(`[Excel Parser] Downloaded ${buffer.byteLength} bytes`);
+
+	// Parse Excel workbook
+	const workbook = XLSX.read(buffer, { type: "array" });
+	console.log(`[Excel Parser] Found sheets: ${workbook.SheetNames.join(", ")}`);
+
+	const sheets: Record<string, CoachRow[]> = {};
+
+	for (const sheetName of workbook.SheetNames) {
+		// FOR TESTING: Only process first sheet
+		if (Object.keys(sheets).length > 0) {
+			console.log(`[Excel Parser] TESTING MODE: Skipping remaining sheets`);
```

Biome (2.1.2)

[error] Do not use template literals if interpolation and special-character handling are not needed.

Safe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)